src/domain/bitcoin/lightning/index.types.d.ts

```diff
@@ -61,30 +80,59 @@ type LnFeeCalculator = {
   max(amount: Satoshis): Satoshis
 }
 
+type PayInvoiceResult = {
+  roundedUpFee: Satoshis
```

Comment:
thinking more about this, is weird that we didn't save the preimage (paymentSecret) in the ledger `ledger.settleLndPayment(id)`. I think should be showed in the app because is the way to do support requests to external services that do not handle the payments correctly. We have had issues with strike, chivo and opennode and the users need to request (to our support chat) the secret/preimage to submit a support case

Reply:
Good idea... it might be easier to make this change directly in the new use-case branch